src/Components/Company/AddNewCompany/addNewCompanyComponent.tsx

```diff
@@ -452,6 +469,31 @@ export const AddNewCompanyComponent = (props: any) => {
                       </Button>
                     </Upload>
                   </Form.Item>
+                  {(companyRole === CompanyRole.MINISTRY ||
+                    companyRoleType === CompanyRole.MINISTRY) && (
```

Comment:
Its better if we can have one state variable to keep track of the company role instead of having multiple variables